src/components/Form.jsx

```diff
@@ -26,98 +28,167 @@ export default function Form() {
+      setVisibleCard('success')
     }
   }
 
+  const handleCloseClick = () => {
+    dispatch(drawerIsOpened(false))
+    resetForm()
+  }
+
   useEffect(() => {
     !isDrawerOpen && resetForm()
   }, [isDrawerOpen])
 
   return (
-    <FormStyled onSubmit={handleSubmit}>
-      <label htmlFor="name">1. Add product name:</label>
-      <input
-        onChange={(event) => setName(event.target.value)}
-        value={name}
-        type="text"
-        id="name"
-        maxLength="40"
-        autoFocus
-        placeholder="E.g. Nivea face cream"
-      />
-      {name.length >= 40 && (
-        <p>The product name can consist of up to 40 characters.</p>
-      )}
-
-      <label htmlFor="date">2. When did you open the product?</label>
-      <input
-        onChange={(event) => setDate(event.target.value)}
-        value={date}
-        type="date"
-        min="2018-01-01"
-        max={currentDate}
-        id="date"
-      />
-
-      <label htmlFor="month">
-        3. In how many months does the product expire?
-      </label>
-      <ContainerStyled>
+    <FormStyled onSubmit={(event) => event.preventDefault()}>
+      <Card isVisible={visibleCard === 'nameInput'}>
+        <label htmlFor="name">1. Add product name:</label>
+        <input
+          onChange={(event) => setName(event.target.value)}
+          value={name}
+          type="text"
+          id="name"
+          maxLength="40"
+          autoFocus
+          placeholder="E.g. Nivea face cream"
+        />
+        {name.length >= 40 && (
+          <p>The product name can consist of up to 40 characters.</p>
+        )}
+
+        <Button text="Next" onClick={() => setVisibleCard('dateInput')}/>
+        <div className="step-indicator">
+          <Circle className="active"/>
+          <Circle/>
+          <Circle/>
+          <Circle/>
+        </div>
+      </Card>
+
+      <Card isVisible={visibleCard === 'dateInput'}>
+        <label htmlFor="date">2. When did you open the product?</label>
+        <input
+          onChange={(event) => setDate(event.target.value)}
+          value={date}
+          type="date"
+          min="2018-01-01"
+          max={currentDate}
+          id="date"
+        />
+        <div className="button-container">
+          <Button
+            text="Back"
+            isCancel
+            onClick={() => setVisibleCard('nameInput')}
+          />
+          <Button text="Next" onClick={() => setVisibleCard('monthInput')}/>
+        </div>
+        <div className="step-indicator">
+          <Circle/>
+          <Circle className="active"/>
+          <Circle/>
+          <Circle/>
+        </div>
+      </Card>
+
+      <Card isVisible={visibleCard === 'monthInput'}>
+        <label htmlFor="month">
+          3. In how many months does the product expire?
+        </label>
+        <ContainerStyled>
+          <input
+            onChange={(event) => setMonth(event.target.value)}
+            value={month}
+            type="number"
```

Comment:
Im Firefox wird das Input Feld type number mit dem Spinner (increment- & decrement Pfeilen) angezeigt, in Chrome nicht. Die Pfeile sind aber super klein. Mit der Maus noch bedienbar, aber mit dem Finger wird es schwer. Vielleicht lässt der sich ja für Firefox verstecken, damit es so aussieht wie in Chrome?
Mobilansicht in Safari und Chrome schaut super aus!  
https://stackoverflow.com/questions/3790935/can-i-hide-the-html5-number-input-s-spin-box 

Ansicht input number in Firefox:
<img width="386" alt="Screenshot 2020-07-23 at 15 43 03" src="https://user-images.githubusercontent.com/37314329/88295570-e3458e00-ccfd-11ea-8e04-48a987165e43.png">

Reply:
Muss ich mir noch mal ansehen. 👀